aml-development/ozp-hud
# AMLNG-700: Undo delete folder

AMLNG-700: Bookmark Folder / un-delete button

Acceptance Criteria:
- ability to restore deleted folder
- undelete button with instructions
- un-delete warning/popup
- notification with link to restore folder

To test:
The `undo_delete_folder` branches are needed from ozp-hud/ozp-center/ozp-backend/ozp-react-commons.

- Log into HUD with any user.
  - If the user does not already have bookmarks & bookmark folders, find listings to bookmark, and create some folders.
- Choose the Delete option from any Folder menu.
Results:
Banner is displayed alerting the user the folder was deleted.
The **undo** link will restore the folder
The banner will go away after 10 seconds if not interacted with
The user can click the X in the banner to close it.

- Refresh the page
Results:
A new notification will be active
Notification will tell the user a folder was recently deleted
The Notification will have a button that allows the user to restore the folder

- Go to Center, Log in as an admin user, if not already.
- Go to Center Settings -> Notifications
Results:
Notification about restoring a deleted folder will NOT be visible in Active Notifications.
